### PR TITLE
Add psalm and GitHub Actions configs to .gitattributes

### PR DESCRIPTION
**Pull request by sidz**
This PR:
- [x] Adds psalm config files to the .gitattributes section
- [x] Adds GitHub Actions config files to the .gitattributes section

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -2,13 +2,10 @@ docs export-ignore
 examples export-ignore
 test export-ignore
 travis export-ignore
-.gitattributes export-ignore
-.gitignore export-ignore
-.travis.yml export-ignore
+/.* export-ignore
```
Comment (kelunik) [inline .gitattributes:5]: It's probably a good idea to always ignore dotfiles, yeah, could you move that to the top with a comment?

Comment (sidz) [reply .gitattributes]: done

**Review by kelunik — DISMISSED** [review]

**Review by kelunik — APPROVED** [review]

**Comment by sidz** [comment]
looks like previous variant of comment in gitattributes is not correct as github highlight it.
docs: https://git-scm.com/docs/gitignore#_pattern_format

**Review by kelunik — APPROVED** [review]

**Comment by kelunik** [comment]
Good point, needs a fix in `byte-stream` as well then.

**Comment by kelunik** [comment]
Thanks!

**Pull request merged**